codersdk/deployment.go

```diff
@@ -1834,6 +1835,15 @@ Write out the current server config as YAML to stdout.`,
 			Group:       &deploymentGroupClient,
 			YAML:        "webTerminalRenderer",
 		},
+		{
+			Name:        "Allow Workspace Renames",
+			Description: "DEPRECATED: Allow users to rename their workspaces. Use only for temporary compatibility reasons, this flag will no longer function after 2024-04-01.",
```

Comment:
The expiration date should probably be stored in codersdk as an exported var so we don't have to duplicate it.